pkg/users/accounts-list.js

```diff
@@ -18,72 +18,281 @@
+        { title: _("Full name"), sortable: true },
+        { title: _("ID"), sortable: true },
+        { title: _("Last active"), sortable: true },
+        { title: _("Group") },
+    ];
+
+    const sortRows = (rows, direction, idx) => {
+        const sortedRows = rows.sort((a, b) => {
+            const aitem = a.columns[idx];
+            const bitem = b.columns[idx];
+            const aname = a.columns[0];
+            const bname = b.columns[0];
+
+            // current user is always first
+            if (aname.sortKey === current_user)
+                return direction === SortByDirection.asc ? -1 : 1;
+            if (bname.sortKey === current_user)
+                return direction === SortByDirection.asc ? 1 : -1;
+            // sorting last login
+            if (idx === 3) {
+                if (aitem.sortKey === "logged in")
+                    return -1;
+                if (bitem.sortKey === "logged in")
+                    return 1;
+                if (aitem.sortKey === "never")
+                    return 1;
+                if (bitem.sortKey === "never")
+                    return -1;
+
+                return bitem.sortKey - aitem.sortKey;
+            }
+
+            return ((typeof aitem == 'string' ? aitem : (aitem.sortKey || aitem.title)).localeCompare(typeof bitem == 'string' ? bitem : (bitem.sortKey || bitem.title)));
+        });
+        return direction === SortByDirection.asc ? sortedRows : sortedRows.reverse();
+    };
+
+    const tableToolbar = (
+        <Toolbar>
+            <ToolbarContent className="accounts-toolbar-header">
+                <ToolbarItem>
+                    <SearchInput id="accounts-filter"
+                                 placeholder={_("Search for name, group or ID")}
+                                 value={currentTextFilter}
+                                 onChange={setCurrentTextFilter}
+                                 onClear={() => setCurrentTextFilter('')} />
+                </ToolbarItem>
+                { superuser.allowed &&
+                    <>
+                        <ToolbarItem variant="separator" />
+                        <ToolbarItem alignment={{ md: 'alignRight' }}>
+                            <Button id="accounts-create" onClick={() => account_create_dialog(accounts)}>
+                                {_("Create new account")}
+                            </Button>
+                        </ToolbarItem>
+                    </>
+                }
+            </ToolbarContent>
+        </Toolbar>
+    );
+
+    return (
+        <Card className="ct-card">
+            <CardHeader>
+                <CardTitle>
+                    <Text component={TextVariants.h2}>{_("Accounts")}</Text>
+                </CardTitle>
+                {tableToolbar}
+            </CardHeader>
+            {filtered_accounts.length === 0
+                ? <EmptyStatePanel title={_("No matching results")} icon={SearchIcon} />
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17830-20221130-151606-eb9be7bf-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)